tests/common.go

```diff
@@ -1207,11 +1371,16 @@ func updateNamespace(in interface{}, namespaceMapping map[string]string) error {
 }
 
 // AfterEachTest runs collect support bundle after each test when it fails
-func AfterEachTest(contexts []*scheduler.Context) {
+func AfterEachTest(contexts []*scheduler.Context, ids ...int) {
```

Comment:
why can't we use []int in this case as I can't see that you loop thru the ...int

Reply:
not all tests are calling AfterEachTest method with testrail Id, with ... we can pass no arguments and it will still work.